src/interactions/scrollInteraction.ts

```diff
@@ -0,0 +1,58 @@
+///<reference path="../reference.ts" />
+
+module Plottable {
+export module Interaction {
+
+  export type ScrollCallback = (p: Point, deltaAmount: number) => any;
+
+  export class Scroll extends Interaction.AbstractInteraction {
+
+    /**
+     * The number of pixels occupied in a line.
+     */
+    public static PIXELS_PER_LINE = 120;
```

Comment:
D3.  Of course not necessarily a good source but I found this number.

Also, this should be commented probably on the ScrollInteraction PR.